src/main/java/codechicken/lib/vec/Cuboid6.java

```diff
@@ -88,8 +92,8 @@ public Cuboid6 add(Vector3 vec) {
         return add(vec.x, vec.y, vec.z);
     }
 
-    public Cuboid6 add(BlockPos pos) {
```

Comment:
This breaks binary compatibility.

Reply:
Well, i need revert back this change? I changed this because BlockPos extends Vec3i

Reply:
The JVM doesn't care about class hierarchy when invoking a method all it knows is its calling method add(Lnet/minecraft/math/blah/BlockPos;) not add(Lnet/minecraft/math/blah/Vec3i;). Add the new method and overload with a BlockPos one, delegate the call in blockPos to the new one with Vec3i.